src/main/java/com/crud/tasks/controller/TaskNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.crud.tasks.controller;
+
+public class TaskNotFoundException extends Exception {
```

Comment:
to jest ok, ale dobrze byloby dodac adnotacje na klasie np
`@ResponseStatus(code = HttpStatus.NOT_FOUND)`

Reply:
pytanie tez czy to musi byc Exception, czy nie wystatczylby RuntimeException